frontend/translations/en.po

```diff
@@ -1999,6 +1999,14 @@ msgstr "Resolved value:"
 msgid "inspect.tabs.switcher.label"
 msgstr "Layer info"
 
+#: src/app/main/ui/inspect/right_sidebar.cljs:165
+msgid "inspect.tabs.switcher.color-space.label"
```

Comment:
MINOR

This translation string has tab and switcher as different segments, but they represent the same semantical section, so is using two segments for represent the same location.

I think it should be `inspect.tabs-switcher.color-space-label`.

In general, translation strings are composed of `<first-level-location>.<second-level-location>.<identifier>` and in very rare case it can have the `<third-level-location>` before the identifier.

Reply:
Translations IDs refactored in a new branch https://github.com/penpot/penpot/pull/7765